src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/controller/MaintainSupplierPricingControllerTest.kt

```diff
@@ -587,6 +587,32 @@ class MaintainSupplierPricingControllerTest(@Autowired private val wac: WebAppli
     verify(service, never()).addPriceException(any(), any(), any(), any(), any(), any())
   }
 
+  @Test
+  internal fun `cannot add an the same price for a  price exception`() {
```

Comment:
`cannot add an exception for the same price?`

Reply:
I think a case of more haste less speed, doh! Done.